.travis.yml

```diff
@@ -0,0 +1,29 @@
+
+# Build matrix
+language: node_js
+
+gemfile:
```

Comment:
Why a gemfile? 🤔

Reply:
doh! yeah that's not needed (I have a default travis config that copy then strip bits out)